paymentsheet/src/test/java/com/stripe/android/paymentsheet/IntentConfirmationHandlerTest.kt

```diff
@@ -1640,26 +1714,21 @@ class IntentConfirmationHandlerTest {
                 )
             )
 
-            googlePayCallbackHandler.onResult(
-                GooglePayPaymentMethodLauncher.Result.Completed(
-                    paymentMethod = PaymentMethodFixtures.CARD_PAYMENT_METHOD,
-                )
-            )
+            googlePayCallbackHandler.onResult(googlePayResult)
+
+            if (googlePayResult !is GooglePayPaymentMethodLauncher.Result.Completed) {
```

Comment:
I think it would be better to not have switching behavior here. Can we pass a value in that tells it what to do?